src/main/resources/com/mathworks/ci/TestResultsViewAction/index.jelly

```diff
@@ -0,0 +1,217 @@
+    <l:layout title="MATLAB Test Results">
+        <l:side-panel>
+            <st:include page="sidepanel.jelly" it="${it.build}" optional="true" />
+        </l:side-panel>
+
+        <l:app-bar title="MATLAB Test Results">
+            <button class="jenkins-button" onclick="window.open('https://matlab.mathworks.com/open/github/v1?repo=${GIT_URL}')" tooltip="View source in MATLAB Online">Open in MATLAB Online</button>
+            <t:help href="https://github.com/jenkinsci/matlab-plugin/blob/master/CONFIGDOC.md" tooltip="MATLAB Plugin Configuration Guide" />
+        </l:app-bar>
+        
+        <l:main-panel>
+            <button id="Total" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Total tests">
+                <p>${it.totalCount}<br></br><br></br>Total Tests</p>
+            </button>
+
+            <button id="Passed" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Passed tests">
+                <p>${it.passedCount}<br>
+                    <l:icon class="symbol-checkmark-circle-outline plugin-ionicons-api icon-lg jenkins-!-success-color"/>
+                </br><br></br>Passed</p>
+            </button>
+
+            <button id="Failed" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Failed tests">
+                <p>${it.failedCount}<br>
+                    <l:icon class="symbol-close-circle-outline plugin-ionicons-api icon-lg jenkins-!-error-color"/>
+                </br><br></br>Failed</p>
+            </button>
+
+            <button id="Incomplete" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Incomplete tests">
+                <p>${it.incompleteCount}<br>
+                    <l:icon class="symbol-alert-circle-outline plugin-ionicons-api icon-lg jenkins-!-warning-color"/>
+                </br><br></br>Incomplete</p>
+            </button>
+
+            <button id="NotRun" onclick="showTests(id)" class="jenkins-button jenkins-!-margin-4 jenkins-!-padding-0" style="display: inline; width:8em;" tooltip="Not run tests">
+                <p>${it.notRunCount}<br>
+                    <l:icon class="symbol-remove-circle-outline plugin-ionicons-api icon-lg"/>
+                </br><br></br>Not Run</p>
+            </button>
+
+            <j:forEach var="testSession" items="${it.testResults}" varStatus="status">
+                <table class="jenkins-table sortable" id="testResultsTable">
+                    <thead>
+                        <tr>
+                            <th class="pane-header" style="width: auto; min-width: 20em;">
+                                Test Name

```

Comment:
Test name should include the procedure name and information about parameterization. How about Test File instead?

Reply:
Good point, updated!